src/pages/workspace/distanceRates/PolicyDistanceRatesPage.tsx

```diff
@@ -0,0 +1,280 @@
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import * as CurrencyUtils from '@libs/CurrencyUtils';
+import type {CentralPaneNavigatorParamList} from '@navigation/types';
+import AdminPolicyAccessOrNotFoundWrapper from '@pages/workspace/AdminPolicyAccessOrNotFoundWrapper';
+import PaidPolicyAccessOrNotFoundWrapper from '@pages/workspace/PaidPolicyAccessOrNotFoundWrapper';
+import {openPolicyDistanceRatesPage} from '@userActions/Policy';
+import ButtonWithDropdownMenu from '@src/components/ButtonWithDropdownMenu';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type SCREENS from '@src/SCREENS';
+import type Policy from '@src/types/onyx/Policy';
+import type {CustomUnit, Rate} from '@src/types/onyx/Policy';
+
+type RateForList = {
+    value: string;
+    text: string;
+    keyForList: string;
+    isSelected: boolean;
+    rightElement: React.ReactNode;
+};
+
+type PolicyDistanceRatesPageOnyxProps = {
+    /** Policy details */
+    policy: OnyxEntry<Policy>;
+};
+
+type PolicyDistanceRatesPageProps = PolicyDistanceRatesPageOnyxProps & StackScreenProps<CentralPaneNavigatorParamList, typeof SCREENS.WORKSPACE.DISTANCE_RATES>;
+
+function PolicyDistanceRatesPage({policy, route}: PolicyDistanceRatesPageProps) {
+    const {isSmallScreenWidth} = useWindowDimensions();
+    const styles = useThemeStyles();
+    const theme = useTheme();
+    const {translate} = useLocalize();
+    const [selectedDistanceRates, setSelectedDistanceRates] = useState<Rate[]>([]);
+    const [isWarningModalVisible, setIsWarningModalVisible] = useState(false);
+    const dropdownButtonRef = useRef(null);
+
+    const customUnit: CustomUnit | undefined = useMemo(
+        () => (policy?.customUnits !== undefined ? policy?.customUnits[Object.keys(policy?.customUnits)[0]] : undefined),
+        [policy?.customUnits],
+    );
+    const customUnitRates: Record<string, Rate> = useMemo(() => customUnit?.rates ?? {}, [customUnit]);
+
+    useEffect(() => {
+        openPolicyDistanceRatesPage(route.params.policyID);
+    }, [route.params.policyID]);
+
+    const distanceRatesList = useMemo<RateForList[]>(
+        () =>
+            Object.values(customUnitRates).map((value) => ({
+                value: value.customUnitRateID ?? '',
+                text: `${CurrencyUtils.convertAmountToDisplayString(value.rate, value.currency ?? CONST.CURRENCY.USD)} / ${translate(
+                    `common.${customUnit?.attributes?.unit ?? CONST.CUSTOM_UNITS.DISTANCE_UNIT_MILES}`,
+                )}`,
+                keyForList: value.customUnitRateID ?? '',
+                isSelected: selectedDistanceRates.find((rate) => rate.customUnitRateID === value.customUnitRateID) !== undefined,
+                rightElement: (
+                    <View style={styles.flexRow}>
+                        <Text style={[styles.alignSelfCenter, !value.enabled && styles.disabledText]}>
```

Comment:
```suggestion
                        <Text style={[styles.alignSelfCenter, styles.disabledText]}>
```

Other pages have it like this, could we make it consistent e.g. https://dev.new.expensify.com:8082/workspace/D9B7E2B075622249/tags

Reply:
This is something I've asked about design team yesterday. We agreed that disabled text should have different color. I will change it to textSupporting specifically.

<img width="790" alt="image" src="https://github.com/Expensify/App/assets/23487612/e010fb0d-2eab-4638-9af6-e7f398268249">

Reply:
Thanks!